fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1349,6 +1351,8 @@ public void saveImage(File curFile, long replayedJournalId) throws IOException {
             checksum = analyzeManager.saveAnalyze(dos, checksum);
             dos.writeLong(checksum);
             checksum = workGroupMgr.saveWorkGroups(dos, checksum);
+            checksum = auth.writeAsGson(dos, checksum);
```

Comment:
You should write the checksum before writing newly added image

Reply:
And read the checksum before reading newly added image

Reply:
Checksum is calculated in write(), so we can only write it afterwards.